tests/shared/core/test_domain.py

```diff
@@ -75,16 +75,16 @@ async def test_create_train_data_no_history(domain: Domain, stories_path: Text):
 
     assert hashed == [
```

Comment:
the comments here and in `test_telemetry` are pretty extensive. What about ignoring the entire files for now?

Reply:
It's better to add a ````flake8: noqa``` or re-add the per-file-ignore at the setup?